plugins/Transitions/javascripts/transitions.js

```diff
@@ -79,7 +79,14 @@ DataTable_RowActions_Transitions.prototype.doOpenPopover = function (link) {
 
     var i = 0;
     while (i < parts.length) {
-        var paramName = decodeURIComponent(parts[i]);
+        var paramName = '';
+
+        try {
+            paramName = decodeURIComponent(parts[i]);
+        } catch (e) {
+            // invalid parameter
```

Comment:
Shouldn't we add a `continue` here? Wouldn't otherwise `overrideParams` end up having an array element with an empty string key? 🤔 
```suggestion
            // invalid parameter
            i += 1;
            continue;
```

Reply:
The continue is already handed by the following check for `ALLOWED_OVERRIDE_PARAMS.indexOf(paramName)`, that one will never succeed with an empty string.

We could add an additional one here for clarity, but it should not change the behaviour.

Reply:
Ah true. Somehow thought `ALLOWED_OVERRIDE_PARAMS` would be a string and an `indexOf` on a string looking for an empty string is always `0` 🙈  All good then.